### PR TITLE
Add missing internal documentation

### PR DESCRIPTION
**Pull request by Hinton**
## Objective

Adds the Advanced Setup, Scim and Tunnel documentation.

### CODE REVIEW

**Review by MGibson1 — DISMISSED** [review]

File: docs/getting-started/server/advanced-setup.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# Advanced Server Setup
+
+Once you have your server up and running, there may be some additional configuration required to
+activate all of Bitwarden's features.
+
+## Premium features
```
[the remark this answers is not in the document]
Comment (Hinton) [reply docs/getting-started/server/advanced-setup.md]: True, can you please verify that all the titles are correctly capitalized now?

File: docs/getting-started/server/advanced-setup.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+Docker compose will spin up a local smtp server that can be used, but it’s also possible to use
+other services such as Mailtrap, or Amazon to debug the amazon integration.
+
+- Amazon Simple Email Service - the user secrets vault item includes a separate attachment called
+  `amazon_key_for_email.json`. Add this key to your user secrets to use Amazon's mail service.
+  Caution: this will send emails to real live email addresses!
+- [bytemark/docker-smtp](https://github.com/BytemarkHosting/docker-smtp) - a local SMTP server
+  running on Docker.
+
+## File Uploads (File Sends and attachments)
```
Comment (withinfocus) [inline docs/getting-started/server/advanced-setup.md:40]: ℹ️ "Attachments".

File: docs/getting-started/server/scim.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+- A [local development server](./guide.mdx)
+- [Web vault](../clients/web-vault/index.mdx)
+- An enterprise organization
+- Mailcatcher or a similar local mailservice so that you don't spam real email addresses with test
+  invites (this is included in the server setup guide)
+
+## Steps
+
+### Enable SCIM for Your Organization
```
Comment (withinfocus) [inline docs/getting-started/server/scim.md:30]: ℹ️ Often times we allow "your" to be lowercase in titles.

File: docs/getting-started/server/scim.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+3. Start the SCIM project in your local server repository:
+
+   ```bash
+   cd bitwarden_license/src/Scim
+   dotnet run
+   ```
+
+4. Verify that the SCIM project started successfully by navigating to `http://localhost:44559/alive`
+
+### Expose your local port
```
Comment (withinfocus) [inline docs/getting-started/server/scim.md:47]: ℹ️ "Local Port".

**Review by withinfocus — CHANGES_REQUESTED** [review]
Since we're fixing!

**Review by withinfocus — APPROVED** [review]

**Pull request merged**